force-app/main/default/reports/UserSetup/License_Summary_CV9.report-meta.xml

```diff
@@ -0,0 +1,616 @@
+    </columns>
+    <columns>
+        <field>Contact.HealthCloudGA__StatusGroup__c</field>
+    </columns>
+    <columns>
+        <field>Contact.HealthCloudGA__Testing_Status__c</field>
+    </columns>
+    <columns>
+        <field>Contact.HealthCloudGA__TotalReferrals__c</field>
+    </columns>
+    <columns>
+        <field>Account.HealthCloudGA__Active__c</field>
+    </columns>
+    <columns>
+        <field>Account.HealthCloudGA__CarePlan__c</field>
+    </columns>
+    <columns>
+        <field>Account.DEC__c</field>
+    </columns>
+    <columns>
+        <field>Account.HealthCloudGA__Disabled__c</field>
+    </columns>
+    <columns>
+        <field>Account.HealthCloudGA__EnrollmentType__c</field>
+    </columns>
+    <columns>
+        <field>Account.HealthCloudGA__IndividualId__c</field>
+    </columns>
+    <columns>
+        <field>Account.HealthCloudGA__IndividualType__c</field>
+    </columns>
+    <columns>
+        <field>Account.HealthCloudGA__Institution__c</field>
+    </columns>
+    <columns>
+        <field>Account.Legacy_External_ID__c</field>
+    </columns>
+    <columns>
+        <field>Account.HealthCloudGA__LowIncome__c</field>
+    </columns>
+    <columns>
+        <field>Account.HealthCloudGA__MedicaidEligibilityStatus__c</field>
+    </columns>
+    <columns>
+        <field>Account.HealthCloudGA__MedicalRecordNumber__c</field>
+    </columns>
+    <columns>
+        <field>Account.HealthCloudGA__MedicareEnrollee__c</field>
+    </columns>
+    <columns>
+        <field>Account.HealthCloudGA__OREC__c</field>
+    </columns>
+    <columns>
+        <field>Account.HealthCloudGA__PayerType__c</field>
+    </columns>
+    <columns>
+        <field>Account.Person_Full_Name__c</field>
+    </columns>
+    <columns>
+        <field>Account.Pharmacy_ID__c</field>
+    </columns>
+    <columns>
+        <field>Account.Pharmacy_Name__c</field>
+    </columns>
+    <columns>
+        <field>Account.HealthCloudGA__PrimaryContact__c</field>
+    </columns>
+    <columns>
+        <field>Account.Record_Type_Name__c</field>
+    </columns>
+    <columns>
+        <field>Account.HealthCloudGA__SourceSystem__c</field>
+    </columns>
+    <columns>
+        <field>Account.HealthCloudGA__SourceSystemId__c</field>
+    </columns>
+    <columns>
+        <field>Account.HealthCloudGA__TaxId__c</field>
+    </columns>
+    <format>Summary</format>
+    <groupingsDown>
+        <dateGranularity>Day</dateGranularity>
+        <field>PROFILE</field>
+        <sortOrder>Asc</sortOrder>
+    </groupingsDown>
+    <groupingsDown>
+        <dateGranularity>Day</dateGranularity>
+        <field>ROLE</field>
+        <sortOrder>Asc</sortOrder>
+    </groupingsDown>
+    <name>License Summary</name>
+    <params>
+        <name>active</name>
+        <value>active</value>
+    </params>
+    <params>
+        <name>co</name>
+        <value>1</value>
+    </params>
+    <reportType>User</reportType>
+    <showDetails>true</showDetails>
```

Comment:
This is a summary report and will not need all the fields listed -- showdetail should be false. @mhussaincgi @RanadheerRG